go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/kyma-project/cli
 go 1.14
 
 replace (
+	github.com/kyma-incubator/hydroform/function => github.com/kfurgol/hydroform/function v0.0.0-20201125121656-f1b81478432b
```

Comment:
Please remove this replace statement and bump the version in the require statement with the origin version.

Reply:
It is a temporary replacement for easier testing the [PR in Hydroform](https://github.com/kyma-incubator/hydroform/pull/144), that's why it's still labeled WIP. After merging Hydroform PR I will of course remove the replacement.

Reply:
Ok, please consider creating draft PRs when not finished, that way reviewers do not get notified to review.